modules/checklist/steps/add-logo.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Elementor\Modules\Checklist\Steps;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Add_Logo extends Step_Base {
+	const STEP_ID = 'add_logo';
+
+	public function get_id() : string {
+		return self::STEP_ID;
+	}
+
+	public function is_absolute_completed() : bool {
+		return $this->wordpress_adapter->has_custom_logo();
+	}
+
+	public function get_title() : string {
+		return esc_html__( 'Add your logo', 'elementor' );
+	}
+
+	public function get_description() : string {
+		return __( 'Let\'s start by adding your logo and filling in the site identity settings. This will establish your initial presence and also improve SEO.', 'elementor' );
+	}
+
+	public function get_cta_text() : string {
+		return esc_html__( 'Go to Site Identity', 'elementor' );
+	}
+
+	public function get_cta_url() : string {
+		$link = $this->wordpress_adapter->get_referer();
```

Comment:
What is the logic here? Why do we need to get the referrer?

Reply:
We need to retrieve the current URL in order to redirect the user to site identity

Reply:
this is the way i found to retrieve it, from $_SERVER was not the best way.  Do you have other suggestion?

Reply:
Answered in Slack